java/src/jmri/jmrit/throttle/ThrottleBundle.properties

```diff
@@ -91,7 +91,7 @@ ExThrottleCleanOnDispose=Clean throttle on window close
 ExThrottleSaveThrottleOnLayoutSave=Save throttles when saving throttle windows layout
 ExThrottleSilentSteal=Silently steal throttle if already used somewhere else
 ExThrottleSilentShare=Silently share throttle if already used somewhere else
-ExThrottleLabelApplyWarning=For new preferences to be fully applied, all throttle windows must be closed and reopened.
+ExThrottleLabelApplyWarning=
```

Comment:
I'm concerned about this. If the `ExThrottleLabelApplyWarning` property is not used, why not remove it?

Reply:
for later use? the related Swing text component is still there.

Reply:
but I can clean up if you think that's better

Reply:
What is the point of having the property `ExThrottleLabelApplyWarning` if it doesn't have any text? Or why is the text of the property removed?

Reply:
the test has been removed because the warning text does not apply any more (new preferences are now applied immediately). I'll remove the related component and this property.

Reply:
Thanks.